test/create-embed.test.js

```diff
@@ -0,0 +1,307 @@
+import document from 'global/document';
+import QUnit from 'qunit';
+import createEmbed from '../src/create-embed';
+
+QUnit.module('create-embed', function(hooks) {
+
+  hooks.beforeEach(function() {
+    this.fixture = document.getElementById('qunit-fixture');
+  });
+
+  hooks.afterEach(function() {
+    this.fixture = null;
```

Comment:
should we empty out the fixture and then null it?

Reply:
QUnit does that for us, I believe.